src/execution/__tests__/variables-test.ts

```diff
@@ -226,7 +226,7 @@ describe('Execute: Handles inputs', () => {
           errors: [
             {
               message:
-                'Argument "input" has invalid value ["foo", "bar", "baz"].',
+                'Argument "input" of type "TestInputObject" has invalid value ["foo", "bar", "baz"].',
```

Comment:
I think this can be really helpful when debugging!

Reply:
I can't take credit for it @leebyron tucked this addition into #3808.

Note that after $3808 lands, this will be:

`Argument "TestType.fieldWithObjectInput(input:)" of type "TestInputObject" has invalid value` and it will be even better.